src/main/scala/com.snowplowanalytics.sauna/observers/AmazonS3Observer.scala

```diff
@@ -53,7 +54,7 @@ class AmazonS3Observer(s3: S3, sqs: SQS, queue: Queue) extends Actor with Observ
 
   def receive: Receive = {
     case filePublished: S3FilePublished =>
-      notify(s"Detected new S3 file [${filePublished.path}]")
+      notify(s"Detected new S3 file [${filePublished.id}]")
```

Comment:
It uses the little helper method [here](https://github.com/snowplow/sauna/blob/34d0dfa0bf7b8c13575ddab2467c47f8248cacac/src/main/scala/com.snowplowanalytics.sauna/observers/Observer.scala#L44-L46) and [here](https://github.com/snowplow/sauna/blob/34d0dfa0bf7b8c13575ddab2467c47f8248cacac/src/main/scala/com.snowplowanalytics.sauna/responders/Responder.scala#L91-L93), not the built-in multithreading function. Though I could rename it to avoid weird conflicts.